src/app/reducers/detail-work-item.reducer.ts

```diff
@@ -19,6 +20,10 @@ export const DetailWorkItemReducer: ActionReducer<DetailWorkItemState> =
         return {...action.payload};
       }
 
+      case WorkItemActions.UPDATE_ERROR: {
+        return {...state};
```

Comment:
Why are you updating the state on error?

Reply:
On work item update error we need to reset the field and there is no way for the component to know if it failed in effects. So we need this for this scenario.